src/ol/layer/BaseVector.js

```diff
@@ -252,9 +252,15 @@ class BaseVectorLayer extends Layer {
    * `setStyle()` without arguments to reset to the default style. See
    * [the ol/style/Style module]{@link module:ol/style/Style~Style} for information on the default style.
    *
-   * If your layer has a static style, you can use "flat" style object literals instead of
-   * using the `Style` and symbolizer constructors (`Fill`, `Stroke`, etc.).  See the documentation
-   * for the [flat style types]{@link module:ol/style/flat~FlatStyle} to see what properties are supported.
+   * If your layer has a static style, you can use [flat style](@link module:ol/style/flat~FlatStyle) object
```

Comment:
```suggestion
   * If your layer has a static style, you can use [flat style]{@link module:ol/style/flat~FlatStyle} object
```

Would be nice if this syntax were less weird.  See https://deploy-preview-14740--ol-site.netlify.app/en/latest/apidoc/module-ol_layer_vector-vectorlayer#setStyle for the rendered doc output for this PR.